src/flint/FlintTypes.jl

```diff
@@ -6693,3 +6840,78 @@ macro fq_default_mpoly_do_op(f, R, a...)
     return res
 end
 
+###############################################################################
+#
+#   Docstrings for the systematically added types in this file
+#
+###############################################################################
+
+module DocstringInfo
+using Markdown
+
+struct RingInfo
+    ring_name :: String
+    latex :: String
+end
+Base.convert(::Type{RingInfo}, (name, latex)::Tuple{String, String}) =
+    RingInfo(name, '$'*latex*'$')
+
+base_rings = Dict{Symbol, RingInfo}(
```

Comment:
Defining a custom struct `RingInfo` for this seems a bit overkill to me? Wouldn't a regular tuple do the job just fine?